integration-tests/test_client.py

```diff
@@ -78,15 +122,24 @@ def test_client_logdir_permissions():
 @pytest.mark.usefixtures("register_subman")
 def test_verify_logrotate_feature(insights_client):
     """
-    Verify that the standard logrotate works for insights-client
-    Ref : https://bugzilla.redhat.com/show_bug.cgi?id=1940267
-
-    Test Steps:
-        1 - Perform register and payload operation to ensure both
-            insights-client.log and insights-client-payload.log files have logs
-        2 - Rotate logs by running logrotate command on CLI for insights-client
-        3 - Verify insights-client.log and insights-client-payload.log size is 0B
-        4 - Verify rotated files are created in log dir by comparing number of log files
+    :id: 5442729f-c6bc-4322-aa17-facd538e9fc3
+    :title: Verify Logrotate feature
+    :description: Verify that the logrotate works properly for insights-client
+    :tier: Tier 1
+    :steps:
+        1. Ensure the logrotate configuration file exists
+        2. Register insights-client and perform pazload operations
+        3. Run the logrotate command
+        4. Verify that 2 new log files were created
+        5. Verify the size of insights-client.log
+        6. Verify the size of insights-client-payload.log
+    :expectedresults:
+        1. The logrotate config file exists
+        2. The insights-client registers successfully and logs are populated
+        3. The logrotate command is executed successfully
+        4. Two new log files were created
+        5. The size is 0B
+        6. The size is 0B
```

Comment:
"the size" ... of what?

Reply:
of insights-client.log and insights-client-payload.log